classes/ActionScheduler_WPCLI_QueueRunner.php

```diff
@@ -38,13 +38,14 @@ public function __construct( ActionScheduler_Store $store = null, ActionSchedule
 	 *
 	 * @author Jeremy Pry
 	 *
-	 * @param int  $batch_size The batch size to process.
-	 * @param bool $force      Whether to force running even with too many concurrent processes.
+	 * @param int    $batch_size The batch size to process.
+	 * @param string $group      The group of actions to claim with this batch.
+	 * @param bool   $force      Whether to force running even with too many concurrent processes.
 	 *
 	 * @return int The number of actions that will be run.
 	 * @throws \WP_CLI\ExitException When there are too many concurrent batches.
 	 */
-	public function setup( $batch_size, $force = false ) {
+	public function setup( $batch_size, $group = '', $force = false ) {
```

Comment:
I had to do a double take at changing the param order here, but as it's unreleased, it is fine (and I like having `$group` as the 2nd param). 👍

Reply:
Yeah, I thought group would be better as the second parameter.